packages/mask/src/plugins/Pets/SNSAdaptor/NormalNFT.tsx

```diff
@@ -0,0 +1,99 @@
+import { makeStyles, useStylesExtends } from '@masknet/theme'
+import { Box } from '@mui/material'
+import { useStyles as boxUseStyles } from './PreviewBox'
```

Comment:
A hook's name should always start with `use`

```suggestion
import { useStyles as useBoxStyles } from './PreviewBox'
```